test/integration/sounds.test.js

```diff
@@ -32,6 +32,7 @@ describe('Working with sounds', () => {
 
         // Delete the sound
         await rightClickText('Meow', scope.soundsTab);
+        await driver.sleep(500); // Wait a moment for context menu; only needed for local testing
```

Comment:
@benjiwheeler is this still needed,  or was it left in by mistake? Seems strange that the unload code would impact the right click actions